workspace.code-workspace

```diff
@@ -75,7 +75,7 @@
   "extensions": {
     "recommendations": [
       "folke.vscode-monorepo-workspace",
-      "vtrois.gitmoji-vscode",
+      "seatonjiang.gitmoji-vscode",
```

Comment:
Is there any difference between both extensions?

Reply:
For me, `vtrois.gitmoji-vscode` is not available. So I guess `vtrois` sold extension recently to `seatonjiang`.

Reply:
Oh, he moved repo to his personal account from the organization account.

Reply:
Oh got ya. Nice catch!